tests/test_filter.py

```diff
@@ -19,122 +19,75 @@ def calendar_event():
 
-    def test_filter_string_exclude_passes(self):
-        """Test setting user agent"""
+    def test_filter_string_exclude_passes(self) -> None:
         filt = Filter("['blue']", "")
         assert filt.filter("summary", "description") is True
 
-    def test_filter_string_exclude(self):
-        """Test setting user agent"""
+    def test_filter_string_exclude(self) -> None:
         filt = Filter("['um']", "")
         assert filt.filter("summary", "description") is False
 
-    def test_filter_string_exclude_is_not_case_sensitive(self):
-        """Test setting user agent"""
+    def test_filter_string_exclude_is_not_case_sensitive(self) -> None:
         filt = Filter("['um']", "")
         assert filt.filter("SUMMARY", "description") is False
 
-    def test_filter_string_exclude_but_string_include(self):
-        """Test setting user agent"""
+    def test_filter_string_exclude_but_string_include(self) -> None:
         filt = Filter("['um']", "['crip']")
         assert filt.filter("summary", "description") is True
 
-    def test_filter_string_exclude_but_regex_include(self):
-        """Test setting user agent"""
+    def test_filter_string_exclude_but_regex_include(self) -> None:
         filt = Filter("['um']", "['/crip/']")
         assert filt.filter("summary", "description") is True
 
-    def test_filter_regex_exclude(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude(self) -> None:
         filt = Filter("['/um/']", "")
         assert filt.filter("summary", "description") is False
 
-    def test_filter_regex_exclude_but_string_include(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude_but_string_include(self) -> None:
         filt = Filter("['/um/']", "['crip']")
         assert filt.filter("summary", "description") is True
 
-    def test_filter_regex_exclude_but_regex_include(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude_but_regex_include(self) -> None:
         filt = Filter("['/um/']", "['/crip/']")
         assert filt.filter("summary", "description") is True
 
-    def test_filter_regex_exclude_ignore_case(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude_ignore_case(self) -> None:
         filt = Filter("['/UM/i']", "")
         assert filt.filter("summary", "description") is False
 
-    def test_filter_regex_exclude_ignore_case_multiline(self):
-        """Test setting user agent"""
+    def test_filter_regex_exclude_ignore_case_multiline(self) -> None:
         filt = Filter("['/CRiPT-$/im']", "")
-        assert (
-            filt.filter(
-                "summary",
-                """descript-
-ion""",
-            )
-            is False
-        )
-
-    def test_filter_regex_exclude_ignore_case_multiline_dotall(self):
-        """Test setting user agent"""
+        assert filt.filter("summary", """descript- ion""", ) is False
```

Comment:
This and the other multiline tests are broken -- the descriptions are no longer spanning multiple lines.  Please format with "black --line-length 79" without any configuration file or other options.

Reply:
absolutely, sorry for messing that up :+1: